compose.yml

```diff
@@ -36,6 +36,7 @@ services:
 
   # AdService
   adservice:
+    image: ${IMAGE_NAME}:${IMAGE_VERSION}-adservice
```

Comment:
This would result in a naming like:
`ghcr.io/open-telemetry/demo:0.1.0-adservice` is that what we want?

I usually see the service name as part of the image name, and after the colon only the version number.
Something like this:
`${IMAGE_NAME}-adservice:${IMAGE_VERSION}` to result in:
`ghcr.io/open-telemetry/demo-adservice:0.1.0`

Reply:
thx your review. more image names are discussed in the issue. #78 
This pr now is a draft.(Test Github actions) The current image name in this PR is a temporary thing,  and the final image_name/docker_registry will be used when the majority in the issue approves it.

Reply:
In general, the version number of the image version should strictly follow semver.
However, in some cases, the version will have some specific information in it.
For example https://hub.docker.com/_/python
has `python:3.9-alpine3.15` `python:3.9-alpine3.16` `python:3.9-slim`